include/tesla.hpp

```diff
@@ -3462,7 +3462,7 @@ namespace tsl {
             SharedThreadData *shData = static_cast<SharedThreadData*>(args);
 
             // To prevent focus glitchout, close the overlay immediately when the power button gets pressed
-            hidsysAcquireSleepButtonEventHandle(&shData->powerButtonPressEvent);
+            hidsysAcquireSleepButtonEventHandle(&shData->powerButtonPressEvent, true);
             eventClear(&shData->powerButtonPressEvent);
```

Comment:
This line is not needed because autoclear bool do it automatically

Reply:
Changed to false since we're doing manual clearing